Dockerfile

```diff
@@ -12,8 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GRPC_HEALTH_PROBE_VERSION=v0.2.0
```

Comment:
Duplicate of below: https://github.com/newrelic/newrelic-istio-adapter/pull/4/files#diff-3254677a7917c6c01f55212f86c57fbfR38